stitches.config.ts

```diff
@@ -316,37 +251,19 @@ export const { styled, css, createTheme, getCssText, globalCss, keyframes, confi
 
 export const utils = config.utils;
 
-export const customColors = (primary: Property.Color) => {
-  const primaryColor = colors[tokenToPropertyName(primary)];
+export const customColors = (primary: PrimaryColor) => {
+  const primaryColor: ColorInfo = {
+    name: primary,
+    value: darkColors[`${primary}9`],
```

Comment:
:warning:  You pass this value to both `darkTheme` and `lightTheme`, shouldn't you switch to `lightColors[...]` for light theme ?

I tested and confirm it breaks some theming related to `primaryColor`, for instance in button:

- Button light theme focused
![image](https://user-images.githubusercontent.com/3367393/140172499-fd32ab6c-f27b-450c-a1b2-a942c4940756.png)